src/OpenTelemetry/Metrics/MetricReader.cs

```diff
@@ -30,10 +30,10 @@ public abstract partial class MetricReader : IDisposable
     {
         private const MetricReaderTemporalityPreference MetricReaderTemporalityPreferenceUnspecified = (MetricReaderTemporalityPreference)0;
 
-        private static Func<Type, AggregationTemporality> cumulativeTemporatlityPreferenceFunc =
+        private static Func<Type, AggregationTemporality> cumulativeTemporalityPreferenceFunc =
```

Comment:
OMG, thanks for catching it!

Reply:
😳 My track record for spelling is not doing so hot lately. Though... I've had to say the "temporality" waaay too many times in the past few weeks. The word deserves to misspelled.